site/content/docs/5.0/components/toasts.md

```diff
@@ -41,12 +41,18 @@ Toasts are as flexible as you need and have very little required markup. At a mi
 </div>
 {{< /example >}}
 
+{{< callout warning >}}
+ The usage of `hide` class to achieve a completely hidden toast got deprecated.
+  However, we still handle the class toggling, till next major version (v6)
```

Comment:
I think there's some phrasing to improve here, but I'll let native english speaker take care of it :D

Reply:
Made a stab at this...hope I captured the gist of it correctly